networkx/algorithms/centrality/betweenness.py

```diff
@@ -120,7 +120,7 @@ def betweenness_centrality(
     if k is None:
         nodes = G
     else:
-        nodes = seed.sample(G.nodes(), k)
+        nodes = seed.sample(sorted(G.nodes()), k)
```

Comment:
Unfortunately, nodes are not sortable in general.Using `sorted` should be fine in tests where we know what the nodes are, but in the package code itself we should use `list`.

Reply:
Yup this was an oversight on my part - should be fixed in 2e2a63e.